credential/pom.xml

```diff
@@ -121,13 +121,11 @@
     <dependency>
       <groupId>com.github.mikegirard</groupId>
       <artifactId>spring-social-salesforce</artifactId>
-      <optional>true</optional>
```

Comment:
We wanted these to be optional, so that the `runtime` module explicitly defines what 3rd party services it supports.